### DISCUSSION TITLE
the way to create two edge between same node

### DISCUSSION THREAD

**Discussion by jaeeewon** [xyflow/xyflow#2811] [Q&A] [answered]
normally, edge is like < ------- > , single edge between nodes.

i wanna use two of edge.

like <======>

in order to display two way of transaction.
(ex. a send 10 to b, b send 20 to a)
like
a-------<----20----<-----|
   |-------->---10---->---b

any of advices will be helpful for me to code
thank you ^--^

**Comment by bcakmakoglu** [comment, marked as answer]
You could use a custom node with two handles on the left/right and use those as two way bindings.
Similar to the [custom node example](https://reactflow.dev/docs/examples/nodes/custom-node/) on the docs.

**Reply by jaeeewon** [reply]
i just mixed numerous edges between nodes into single edge by display at once.
thank you for anwsering me

**Reply by GiorgosSpil** [reply]
Hello, @Pleizz . Can you , in a nutshell , tell me how you did that?
